tests/unit/core/reducers/test_addons.js

```diff
@@ -82,4 +82,38 @@ describe('addon reducer', () => {
       },
     });
   });
+
+  it('does not use description from theme_data', () => {
+    // See: https://github.com/mozilla/addons-frontend/issues/2569
+    // TODO: Remove this when
+    // https://github.com/mozilla/addons-frontend/issues/1416 is fixed.
+    const type = ADDON_TYPE_THEME;
+    const state = addons(originalState, {
+      payload: {
+        entities: {
+          addons: {
+            baz: {
+              description: null,
+              slug: 'baz',
+              id: 42,
+              theme_data: { theme_thing: 'some-data', description: 'None' },
+              type,
+            },
+          },
+        },
+      },
+    });
+    expect(state).toEqual({
```

Comment:
I think checking the whole state is overkill and will introduce a maintenance burden. Can you try to only assert the `description` property? I think this will do it but make sure it fails without your fix:

```js
expect(state).toEqual(expect.objectContaining({
  baz: { description: null },
})).
```